From: Sascha Hauer <s.hauer@pengutronix.de>
Date: Fri, 21 Aug 2020 06:21:46 +0200
Subject: [PATCH] ARM: imx8m: Make uart setup function name more generic

So far all i.MX8M variants have the same UART base addresses, so let's
be optimistic that it stays like that and rename imx8mq_uart_setup() to
imx8m_uart_setup().

Signed-off-by: Sascha Hauer <s.hauer@pengutronix.de>
---
 arch/arm/boards/nxp-imx8mm-evk/lowlevel.c | 2 +-
 arch/arm/mach-imx/include/mach/debug_ll.h | 2 +-
 include/serial/imx-uart.h                 | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boards/nxp-imx8mm-evk/lowlevel.c b/arch/arm/boards/nxp-imx8mm-evk/lowlevel.c
index 082aefb8c1c5..96b78d2c6695 100644
--- a/arch/arm/boards/nxp-imx8mm-evk/lowlevel.c
+++ b/arch/arm/boards/nxp-imx8mm-evk/lowlevel.c
@@ -32,7 +32,7 @@ static void setup_uart(void)
 
 	imx8mm_setup_pad(IMX8MM_PAD_UART2_TXD_UART2_TX | UART_PAD_CTRL);
 
-	imx8mq_uart_setup((void *)MX8M_UART2_BASE_ADDR);
+	imx8m_uart_setup((void *)MX8M_UART2_BASE_ADDR);
 
 	if (IS_ENABLED(CONFIG_DEBUG_LL)) {
 		imx8m_uart_setup_ll();
diff --git a/arch/arm/mach-imx/include/mach/debug_ll.h b/arch/arm/mach-imx/include/mach/debug_ll.h
index f2114d50e91e..3217fc1c32f6 100644
--- a/arch/arm/mach-imx/include/mach/debug_ll.h
+++ b/arch/arm/mach-imx/include/mach/debug_ll.h
@@ -105,7 +105,7 @@ static inline void imx8m_uart_setup_ll(void)
 {
 	void *base = IOMEM(IMX_UART_BASE(IMX_DEBUG_SOC,
 					 CONFIG_DEBUG_IMX_UART_PORT));
-	imx8mq_uart_setup(base);
+	imx8m_uart_setup(base);
 }
 
 static inline void PUTC_LL(int c)
diff --git a/include/serial/imx-uart.h b/include/serial/imx-uart.h
index 23cee26fcfb0..c0a03ac05401 100644
--- a/include/serial/imx-uart.h
+++ b/include/serial/imx-uart.h
@@ -201,7 +201,7 @@ static inline void imx7_uart_setup(void __iomem *uartbase)
 	imx_uart_setup(uartbase, 24000000);
 }
 
-static inline void imx8mq_uart_setup(void __iomem *uartbase)
+static inline void imx8m_uart_setup(void __iomem *uartbase)
 {
 	imx_uart_setup(uartbase, 25000000);
 }
